types/textdiff-create/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for textdiff-create 1.0
+// Project: https://github.com/icflorescu/textdiff-create#readme
+// Definitions by: Joe Barnett <https://github.com/ensconced>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace TextDiffCreate {
+    enum ChangeType {
+        DELETE = -1,
+        INSERT = 1,
+        EQUAL = 0,
```

Comment:
underlying types are using const here:
https://github.com/jhchen/fast-diff/blob/master/diff.d.ts#L10-L12
Can you align? TS advertises enums no more:
https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums